specifyweb/frontend/js_src/lib/components/Attachments/index.tsx

```diff
@@ -223,16 +236,18 @@ function Attachments(): JSX.Element {
           </div>
         </Label.Inline>
         <span className="-ml-2 flex-1" />
-        <Label.Inline>
-          {attachmentsText.scale()}
-          <Input.Generic
-            max={maxScale}
-            min={minScale}
-            type="range"
-            value={scale}
-            onValueChange={(value) => setScale(Number.parseInt(value))}
-          />
-        </Label.Inline>
+        {displayScale && (
```

Comment:
For the future, when you do a change like this (i.e, removing some feature or any other change the reasons for which are not obvious without context) please include a brief explanation for why this was done.
Do that either in the commit message, or leave a comment in the code

As a quick hack, if the fix is in response to a particular github issue/command, instead of providing a full explanation of the issue you could just say `Fixes <link to comment or #issueNumber>`